pages/common/license.md

```diff
@@ -8,7 +8,7 @@
 
 - Create a license with custom filename:
 
-`license -o {{filename.txt}} {{license_name}}`
+`license -o {{file}} {{license_name}}`
```

Comment:
Same as above: if this only accepts filenames (not full paths), I believe "filename" makes that clearer.